blog/_posts/2016-10-16-B-More-Stunners-Over-and-Out.md

```diff
@@ -0,0 +1,69 @@
+This summer we had the opportunity to work on the Bundler project alongside supervisor André Arko. We were tasked with the project of making modification to Bundlers’ metrics dashboard so that it can accurately reflect its current metrics. 
+
+
+### Here’s a little short summary of what we’ve accomplished:
+- A better understanding of how Bundler and Rubygems.orgs works
+- Submitted a Pull Request to Rubygems.org server 
+- Built a class that focuses on capturing system information and the options run with Bundler upon installation
+- Refactoring our code…followed by a little more refactoring 
+- Attending Strange Loop Conference and meeting team [Team JaM](https://twitter.com/rgsocJaM).
+
+
+![newfriends1](/img/blog/2016/strangeloop.jpg)
+<br/>
+<font color="grey"><small><i>Ore and [Nicola Hughes](https://twitter.com/GirlMeetsCode) at the Strange Loop Conference</i></small></font>
+
+
+
+
+
+
+![newfriends2](/img/blog/2016/team jam.jpg)
+<br/>
+<font color="grey"><small><i> Team Jam and Ore at the Strange Loop Conference.</i></small></font>
+
+
+
+
+
+### Future plans:
+*Ashley*  - I fully attend to continue working on Bundler/Rubygems.org. There is still a lot of work that needs to be completed within the metrics project! Additionally, I will be starting my new job as a Junior Developer at a local tech company located in Baltimore. :D 
+
+*Ore* - I'm more motivated than ever to finish the project. I got a lot further on the project than I ever thought, which has been a huge boost of confidence. 
+
+
+### Advice:
+*Ashley* - My biggest advice would be to continue to remind yourself that you CAN and you WILL do it! You were chosen to be apart of this fellowship for a reason, so don’t doubt your awesomeness. It’s okay to have bad days. Just remind yourself that no one was born a developer. Every great developer had to go through this learning process at one point to succeed :) 
+
+
+*Ore* - My advice to next year’s participants is to keep on going no matter what. Also, be sure to get lost in the Rails Girls Summer Of Code slack channels; you never know what you'll find in there.
+
+
+
+### Special Thanks:
+Our sincere thank you goes out to our amazing coaches and mentors ( Benedikt 
+André , Lynn, Jamie, Mira and Beverly)  who dedicated their time this summer to help us on this journey. We could not have done it without your ongoing support, mentorship and pep talks! 
+
+
+Lastly, We would like to thank the wonderful organizers of Rails Girls Summer Of Code. Thank you for making this fellowship possible and giving us the opportunity to be a part of it! Because of this program we have grown as developers and individuals. 
+
+
+And for that we are eternally grateful <3 
+
+![Dance](/img/blog/2016/dino.gif)
```

Comment:
please add an image credit to this (e.g. source of the gif if you found it on the internet)